templates/base.html

```diff
@@ -7,10 +7,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description"
       content="{% block meta_description %}Canonical produces Ubuntu, provides commercial services for Ubuntu’s users, and works with hardware manufacturers, software vendors and cloud partners to certify Ubuntu.{% endblock %}" />
+    <meta name="copydoc" content="{% block meta_copydoc %}https://drive.google.com/drive/u/1/folders/0B8feV0jqaac3TmExU3NDcHhFTGc{% endblock %}">
```

Comment:
Not sure this link is correct:
- it shouldn't have `/u/1` in the URL, that's only there if you have multiple Google accounts (in this case '1' selects the second account you're logged in with, which may not be a user's Canonical account).
- https://drive.google.com/drive/folders/0B8feV0jqaac3TmExU3NDcHhFTGc links to a folder, is that expected?

Reply:
@sowasred2012 yes, that's default in case no link is provided for a specific page.

Reply:
@sowasred2012 ahh, I see what you say. will amend it.